packages-content-model/roosterjs-content-model-types/lib/format/formatParts/MarginBlockFormat.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Format of margin-block
+ */
+export type MarginBlockFormat = {
```

Comment:
What will happen if both margin-block-start and margin-top are specified with different values?